src/landing_game_object.py

```diff
@@ -19,6 +22,8 @@ def __init__(
             pos (Vec2d, optional): 2d Position in pixel. (0,0) represents left top corner. Defaults to Vec2d().
         """
         super().__init__()
+        self.ID = self.ID_generator.assign_ID()
```

Comment:
According to pep 8 style convention, uppercase names are used for constants on module level. Since id varies over instances  it should be lowercase